raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -45,53 +45,60 @@ library NettingChannelLibrary {
     }
 
     modifier stillTimeout(Data storage self) {
-        if (self.closed + self.settleTimeout < block.number)
+        if (self.closed + self.settle_timeout < block.number)
             throw;
         _;
     }
 
     modifier timeoutOver(Data storage self) {
-        if (self.closed + self.settleTimeout > block.number)
+        if (self.closed + self.settle_timeout > block.number)
             throw;
         _;
     }
 
     modifier notClosingAddress(Data storage self) {
-        if (msg.sender == self.closingAddress)
+        if (msg.sender == self.closing_address)
             throw;
         _;
     }
 
     /// @notice deposit(uint) to deposit amount to channel.
     /// @dev Deposit an amount to the channel. At least one of the participants
     /// must deposit before the channel is opened.
-    /// @param callerAddress (address) the address of the invoker of the function
-    /// @param channelAddress (address) the address of the channel
+    /// @param caller_address (address) the address of the invoker of the function
+    /// @param channel_address (address) the address of the channel
     /// @param amount (uint) the amount to be deposited to the address
     /// @return success (bool) if the transfer was successful
     /// @return balance (uint256) the new balance of the invoker
-    function deposit(Data storage self, address callerAddress, address channelAddress, uint256 amount) returns (bool success, uint256 balance) {
+    function deposit(
+        Data storage self,
+        address caller_address,
+        address channel_address,
+        uint256 amount
+    )
+        returns (bool success, uint256 balance)
```

Comment:
same here, returns right next to the parentheses to show it's part of the declaration and not the body.